main/src/main/java/cgeo/geocaching/unifiedmap/UnifiedMapActivity.java

```diff
@@ -212,6 +212,7 @@ public void onCreate(final Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
 
         HideActionBarUtils.setContentView(this, R.layout.unifiedmap_activity, true);
+        runInitAndMaintenance(savedInstanceState);
```

Comment:
Wouldn't it be possible to place this line in `AbstractBottomNavigationActivity#onCreate` so we don't need to call it everywhere separately?

Reply:
Thinking behind that was that the calling activity should be initialized before calling init tasks, specifically setContentView.

Reply:
Then `onStart` maybe?

![](https://developer.android.com/guide/components/images/activity_lifecycle.png)

https://developer.android.com/guide/components/activities/activity-lifecycle

Reply:
Good idea. I needed to split up the routine, though, as creating backup tools requires a `savedInstanceState`, which is available in `onCreate`, but not in `onStart`. PR is updated.